package.json

```diff
@@ -39,38 +39,40 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "run-s -l build:*",
+    "build": "run-s -l \"build:*\"",
     "build:clean": "rm -rf dist/ dist-cjs/",
     "build:mjs": "tsc --project tsconfig.build.json",
     "build:cjs": "tsc --project tsconfig.cjs.build.json",
     "build:copy": "cp package.cjs.json dist-cjs/package.json",
-    "test": "run-p -cl test:*",
+    "test": "run-p -cl \"test:*\"",
     "test:ts": "tsc --noEmit",
     "test:eslint": "eslint --color src",
     "test:jest": "jest --colors",
-    "test:prettier": "prettier -l 'src/**/*'",
-    "prettier": "prettier 'src/**/*' --write",
+    "test:prettier": "prettier -l \"src/**/*\"",
+    "prettier": "prettier \"src/**/*\" --write",
     "jest": "jest --colors --watch",
     "prepublishOnly": "run-s -l test build"
   },
   "dependencies": {
     "colorette": "^2.0.19"
   },
   "peerDependencies": {
-    "zod": "^3.13.2"
+    "zod": "^3.24.2"
   },
   "devDependencies": {
-    "@types/jest": "^29.5.4",
-    "@types/node": "^16.18.24",
-    "eslint": "^8.48.0",
-    "eslint-config-lostfictions": "^6.0.0",
+    "@types/jest": "^29.5.14",
+    "@types/node": "^22.13.10",
+    "@typescript-eslint/utils": "^8.27.0",
+    "eslint": "^9.22.0",
+    "eslint-config-lostfictions": "^7.0.0-beta.0",
+    "eslint-import-resolver-typescript": "^4.2.2",
```

Comment:
`@typescript-eslint/utils` and `eslint-import-resolver-typescript` shouldn't be necessary, with the flat config format eslint configs can bring in their own dependencies

Reply:
Yeah, I had to add them, because yarn would complain otherwise, because they are implicitly pulled in by other debs, if we switch to pnpm that might not be a problem.